forch/authenticator.py

```diff
@@ -33,7 +33,12 @@ def __init__(self, auth_config, auth_callback=None, radius_query_object=None):
         radius_info = auth_config.radius_info
         radius_ip = radius_info.server_ip
         radius_port = radius_info.server_port
-        secret = radius_info.secret
+        secret = None
+        if radius_info.radius_secret_handler:
+            secret = os.popen(radius_info.radius_secret_handler).read().strip()
+        # presence of secret field overrides any secret retreived from handler
+        if radius_info.secret:
```

Comment:
this should be removed -- there never should be an explicit secret in the config file -- no need to specify two ways of doing the same thing.